alws/alembic/versions/e952dc755f9d_add_team_to_test_repos.py

```diff
@@ -0,0 +1,164 @@
+"""Add Team to Test repos
+
+Revision ID: e952dc755f9d
+Revises: 07dad1dc5105
+Create Date: 2024-05-24 15:58:53.153438
+
+"""
+from alembic import op
+import sqlalchemy as sa
+from sqlalchemy.dialects import postgresql
+
+# revision identifiers, used by Alembic.
+revision = 'e952dc755f9d'
+down_revision = '07dad1dc5105'
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.create_table('test_repository_role_mapping',
+    sa.Column('test_repository_id', sa.Integer(), nullable=False),
+    sa.Column('role_id', sa.Integer(), nullable=False),
+    sa.ForeignKeyConstraint(['role_id'], ['user_roles.id'], name='fk_test_repositories_role_mapping_role_id', ondelete='CASCADE'),
+    sa.ForeignKeyConstraint(['test_repository_id'], ['test_repositories.id'], name='fk_test_repositories_role_mapping_product_id', ondelete='CASCADE'),
+    sa.PrimaryKeyConstraint('test_repository_id', 'role_id')
+    )
+    op.alter_column('build_releases', 'status',
```

Comment:
We need to figure out why all these `nullable=False` are getting introduced in the migration. I bet is a side effect from recent migration to fastapi-sqla. @Korulag, any ideas?

Reply:
Ok, this seems to be a side-effect of the recent migration to sqlalchemy 2. I checked one by one in production db and all can be safely updated to set `nullable` to `False`, but some of them need to be handled:
* `build_tasks.mock_options`: these are old builds from 2021 - I'd say that we can update these with an empty `{}` or, if we don't want to, turn this into `nullable=True`
* `builds.mock_options`: these are old builds from 2021-2022 - I'd say that we can update these with a `{'definitions': {}}` or, if we don't want to, make it `nullable=True`
* `builds.cancel_testing ` - These are builds before the introduction of cancel_testing - can set those without cancel_testing with default value (False) 

@Korulag any thoughts? Also, would you like to make these adjustments within this PR? Or do you prefer to create another one?